### DISCUSSION TITLE
Batch export audio?

### DISCUSSION THREAD

**Discussion by tuskinekinase** [DrCoffey/DeepSqueak#180] [Ideas]
Hello,
I've been using and loving Deepsqueak for marmoset call analysis. Since we often work with calls in the range of several hundreds, I wonder if it's possible to batch export audio clips and spectrograms, instead of doing it one by one as the program now allows in the import/export menu?
Thank you!

**Comment by DrCoffey** [comment]
Hey @emilyhanxiao, there is currently no way of batch exporting audio and spectrograms, only the call stats. I don't think it would be terribly hard to code though.
